test/relations.test.js

```diff
@@ -3629,6 +4828,7 @@ describe('relations', function () {
           });
           cb(null, summary);
         });
+        return cb.promise;
       };
 
       summarize.shared = true; // remoting
```

Comment:
@bajtos I agree... I did do that style in several places, but I missed a few (rewriting these tests is mind-numbing).  
Also, there are a few that can't be written that way, due to the second `then` block needing access to the scope of the first. (for instance, when that second `then` refers to `cat` )